node/flags/flags.go

```diff
@@ -439,6 +421,12 @@ var (
 		Value:    0.05,
 		EnvVar:   common.PrefixEnvVar(EnvVarPrefix, "LITT_DB_READ_CACHE_SIZE_FRACTION"),
 	}
+	LittDBStoragePathsFlag = cli.StringSliceFlag{
+		Name:     common.PrefixFlag(FlagPrefix, "litt-db-storage-paths"),
+		Usage:    "Comma separated list of paths to store the LittDB data files. At least one path must be provided.",
+		Required: true,
+		EnvVar:   common.PrefixEnvVar(EnvVarPrefix, "LITT_DB_STORAGE_PATHS"),
```

Comment:
What's the behavior of changing this flag / what is supported? Some example scenarios:

1. Running with 1 path and I add another path.
2. Running with 2 paths and I remove a path
3. Running with 1 path then i change that path.

Reply:
Can this be optional and fallback to legacy `NODE_DB_PATH` if not set?
Currently the `NODE_DB_PATH=/data/operator/db` and is mounted via docker volume defined from `NODE_DB_PATH_HOST=${EIGENDA_HOME}/db`. Is the migration as simple as setting `NODE_LITT_DB_STORAGE_PATHS=/data/operator/db` or does this path expect something like `NODE_LITT_DB_STORAGE_PATHS=/data/operator/db/chunks_v2` when migrating?

Reply:
@pschork I've added the requested fallback. If you are still using it in a year, I'm going to rugpull you. 😜 

```
	if len(config.LittDBStoragePaths) == 0 {
		logger.Warnf("WARNING: setting NODE_DB_PATH is deprecated and will be removed in a future version. " +
			"Please use NODE_LITT_DB_STORAGE_PATHS instead.")
		config.LittDBStoragePaths = []string{
			config.DbPath + "/chunk_v2_litt",
		}
	}
```